SeasClick.cpp

```diff
@@ -167,9 +157,9 @@ zend_module_entry SeasClick_module_entry =
     SEASCLICK_RES_NAME,
     SeasClick_functions,
     PHP_MINIT(SeasClick),
-    PHP_MSHUTDOWN(SeasClick),
```

Comment:
why remove shutdown callback?

Reply:
It doesn't do anything, so recommended approach is to not invlude PHP_MSHUT / PHP_MINIT unless they do something